src/middlewared/middlewared/plugins/catalogs_linux/apps.py

```diff
@@ -8,7 +8,7 @@ class AppService(Service):
     class Config:
         cli_namespace = 'app'
 
-    @filterable
+    @filterable(roles=['READONLY'])
```

Comment:
I'm sorry but I've been confusing on my review. I meant to add `APP_READ` to `READONLY` and then make these all `roles=['APP_READ']`